features/features.go

```diff
@@ -671,14 +631,6 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						mustRegister()
 
-	FeatureGateUserNamespacesSupport = newFeatureGate("UserNamespacesSupport").
-						reportProblemsToJiraComponent("Node").
-						contactPerson("haircommander").
-						productScope(kubernetes).
-						enhancementPR("https://github.com/kubernetes/enhancements/issues/127").
-						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-						mustRegister()
-
```

Comment:
@haircommander confirmed on Slack that this should be dropped

Reply:
+1

Reply:
Is it locked on upstream, or just on by default? 

We may instead want to promote the feature to GA here and retain the ability to disable it if we find issues, for at least 1 more release